workflow/rules/generate_output.smk

```diff
@@ -72,6 +72,7 @@ checkpoint rki_filter:
     params:
         min_identity=config["quality-criteria"]["min-identity"],
         max_n=config["quality-criteria"]["max-n"],
+        samples_file="config/pep/samples.csv",
```

Comment:
The sample sheet DataFrame object can be directly addressed by using `pep.sample_table`. Please use this, instead of a hardcoded path.

Instead of handing over a whole DataFrame you could use `zip`, `get_samples_for_date`and a new function: `get_include_flag_for_date`. Similar to this: https://github.com/IKIM-Essen/uncovar/blob/master/workflow/scripts/plot-variants-over-time.py#L40